src/api/mock-interceptor.ts

```diff
@@ -95,6 +96,7 @@ export const interceptRequestsOnMock = () => {
   mock.onGet(new RegExp(`${PublicationsApiPaths.FOR_APPROVAL}/*`)).reply(200, mockMyPublications);
 
   //PUBLICATION
+  mock.onGet(new RegExp(`${PublicationsApiPaths.PUBLICATION}/new`)).reply(200, emptyPublication);
```

Comment:
finnes ikke noe endpoint for get new publication?

Reply:
det er kun for bruk på den midlertidige knappen, slik at man får opp skjemaet for testenes skyld. det skal bort når vi fjerner knappen snart.

Reply:
haha, vi sender en get på /new til backend som bare virker med mock=true ja, nais :D

Reply:
eller virker det med mock=false med deg? Jeg får 400 og kommer tilbake til /publication

Reply:
ja, det er kanskje ikke så bra nei. kan omringe de med `if useMockData` da? men det er jo litt dumt. men den knappen skal jo uansett fjernes når vi får testbrukere. kanskje vi bare skal gå løs på det arbeidet nå? oppdatere tester og fjerne knappen?

Reply:
jeg bruker aldri den knappen lenger, så vi kan godt lage en sak på det i hvert fall.
så kan det bare være sånn for min del

Reply:
ok, da gjør jeg det 👍